Spiegel Online.js

```diff
@@ -297,21 +278,20 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [],
-				"seeAlso": [],
+				"date": "07.04.2012",
```

Comment:
we should normalize dates. I just tested and the (current) translator imports todays articles as May 3rd rather than March 5th on my en-US setup. 
Might be worthwhile to look at either a new or an improved utilities function since we'll want to be doing this almost universally.